CefGlue.Common/build/CefGlue.Common.props

```
@@ -15,6 +15,10 @@
         <CefGlueTargetPlatform Condition="'$(Platform)' == 'ARM64'">osx-arm64</CefGlueTargetPlatform>
         <CefGlueTargetPlatform Condition="'$(CefGlueTargetPlatform)' == ''">osx-arm64</CefGlueTargetPlatform> <!-- fallback -->
     </PropertyGroup>
+
+    <PropertyGroup Condition="'$(RuntimeIdentifier)' == '' and $([MSBuild]::IsOSPlatform('Linux'))">
+        <CefGlueTargetPlatform Condition="'$(Platform)' == 'x64'">linux-x64</CefGlueTargetPlatform>
```

since its the only target for linux
```suggestion
        <CefGlueTargetPlatform>linux-x64</CefGlueTargetPlatform>
```

---

So we are not planning any Linux ARM64 support?